src/java.net.http/share/classes/java/net/http/HttpRequest.java

```diff
@@ -303,6 +303,48 @@ protected HttpRequest() {}
         return new HttpRequestBuilderImpl(uri);
     }
 
+    /**
+     * Creates a {@code Builder} seeded from a {@code HttpRequest}.
+     *
+     * <p> This method can be used to build a new request equivalent to the
+     * given request, but with some parts of its state altered.
+     *
+     * @param request the request
+     * @return a new request builder
+     * @throws IllegalArgumentException if a new builder cannot be seeded from
+     *         the given request (for instance, if the request contains illegal
+     *         parameters)
+     * @since TBD
```

Comment:
`@since 16`

we can change it again if it doesn't make the cut

Reply:
Release number added in commit https://github.com/openjdk/jdk/pull/1059/commits/7928083ae954c44b5357064b86af0e1a3f53588b